examples/pnpm-lock.yaml

```diff
@@ -0,0 +1,26 @@
+lockfileVersion: 5.3
```

Comment:
nit: we should tell prettier give up on formatting lock files.

Reply:
agreed, we've done that in some other repos